app/app-config/segmentsConfig.js

```diff
@@ -0,0 +1,35 @@
+define([], function() {
+/* eslint-enable max-statements */
+
+    var segmentData = [
+        {
+            id: 'bikes',
+            url: '/',
+            items: [
```

Comment:
Would it be more explicit to name this key `segments`?  Make the mental mapping a bit easier?